stubs/docker/docker/api/build.pyi

```diff
@@ -1,39 +1,53 @@
-from _typeshed import Incomplete
+from collections.abc import Generator
+from logging import Logger
+from typing import IO, Any, TypedDict
+from typing_extensions import NotRequired
 
-log: Incomplete
+log: Logger
+
+class _ContainerLimits(TypedDict):
+    memory: NotRequired[int]
+    memswap: NotRequired[int]
+    cpushares: NotRequired[int]
+    cpusetcpus: NotRequired[str]
+
+class _Filers(TypedDict):
+    dangling: NotRequired[bool]
+    until: NotRequired[str]
 
 class BuildApiMixin:
     def build(
         self,
-        path: Incomplete | None = None,
+        path: str | None = None,
         tag: str | None = None,
         quiet: bool = False,
-        fileobj: Incomplete | None = None,
+        fileobj: IO[Any] | None = None,
```

Comment:
Tracing it, this will only accept `IO[bytes]` or explicitly `StringIO` (needs import for `StringIO`):

```suggestion
        fileobj: StringIO | IO[bytes] | None = None,
```